integration_test/third_party_apps_data/applications/hbase/centos_rhel/install

```diff
@@ -0,0 +1,36 @@
+set -e
+
+sudo yum -y install \
+    java-1.8.0-openjdk java-1.8.0-openjdk-devel wget
+
+sudo mkdir /opt/hbase
+wget http://apache.mirror.gtcomm.net/hbase/stable/hbase-2.4.9-bin.tar.gz
+sudo tar -xvf hbase-2.4.9-bin.tar.gz -C /opt/hbase --strip 1
+
+JAVA_HOME="$(dirname "$(dirname "$(readlink "$(readlink "$(which javac)")")")")"
+
+cat <<EOF | sudo tee /etc/systemd/system/hbase.service
+[Unit]
+Description=HBase Hadoop database
+After=network.target
+
+[Service]
+Type=forking
+Environment=JAVA_HOME=$JAVA_HOME
+ExecStart=/opt/hbase/bin/start-hbase.sh
+ExecStop=/opt/hbase/bin/stop-hbase.sh
+User=root
+Group=root
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+cat <<EOF | sudo tee /opt/hbase/conf/hbase-env.sh
```

Comment:
Unnecessary use of `cat`. This can just be `sudo tee /opt/hbase/conf/hbase-env.sh <<EOF`.

Reply:
Updated